content/spin/managing-cache.md

```diff
@@ -0,0 +1,77 @@
+title = "Managing Cache"
```

Comment:
We have a bunch of caches - I'd strongly suggest clarifying this as the registry cache

Reply:
Thanks for the review @itowlson, great feedback.

Is this the only user-manageable cache? If so we can update the page name and the page content to use the more specific term "registry" for clarity. However, if there are other caches that are also user-manageable then perhaps we can leave the page name as it is and then separate each of the different caches into their own headings on this page. This content would go under its own heading and refer only to the registry cache.

Reply:
There are a couple of caches in the plugins stuff, and the login token cache - not sure if there are others.  And some other stuff Spin puts on your system but we don't call it caches, e.g. templates.

I am not sure what you mean by "user manageable" in this context... the registry cache is not really "user manageable" except for you can delete it _grin_, and the plugins and login token caches have the same level of manageability.  With all of these, really, there's not much to say except "this is where it is, this is the impact if you delete it," but agree, if we are going to cover other caches, then a separate page makes sense... but I guess I am trying to get a clearer sense of the purpose... like is it tuning or tidying or "How to Obliterate Spin From Your System With Extreme Prejudice"?  I am kind of thinking what is the _task_ or _goal_ that we are trying to answer, if that makes sense...?

Reply:
Completely makes sense. How about a "Spin Cache" menu item in the "Advanced" section, and on that page, we explicitly refer to this PRs content as registry cache (which, as you say, it is)? 

The additional cache can go into that same "Advanced" area as its documentation is rolled out.

Reply:
Please check the latest commit for updates.